README.md

```diff
@@ -33,31 +33,25 @@
       ios % pod install
       ```
 
-   - For Android development, you'll need to set up an earlier version of the Java Development Kit (JDK), which you can do via [Homebrew](https://brew.sh)
+   - For Android development, first install Android Studio: https://developer.android.com/studio
 
+      For Mac OS X, configure your shell environment to reference the newly installed
+      developer tools. In the standard Mac OS Terminal, edit (or create, if needed)
+      your .bash_profile in your home directory and add the following (assuming
+      default installation locations):
 
       ```
-      brew tap AdoptOpenJDK/openjdk
-      brew cask install adoptopenjdk8
+      export JAVA_HOME=/Applications/Android\ Studio.app/Contents/jre/jdk/Contents/Home/
```

Comment:
@ericouyang does this alternative Android setup work for you?

Reply:
I think I finally found the Android setup instructions you followed on the React Native site, will update to include instructions on how to find it... I think they're a little bit out of date (i.e. ANDROID_HOME is technically a deprecated env var, now replaced with ANDROID_SDK_ROOT...)...